apps/metadata-editor/src/app/dashboard/search-filters/search-filters.component.spec.ts

```diff
@@ -0,0 +1,22 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing'
+import { SearchFiltersComponent } from './search-filters.component'
+import { MockBuilder } from 'ng-mocks'
+
+describe('SearchFiltersComponent', () => {
```

Comment:
It's always a bit tricky to decide what to test, but I would at least test that a mock searchConfig with multiple elements is correctly read.

Reply:
You are right, I thought it was already covered by the e2e tests, but it is actually not. I've added a UT for it.